aster.go

```diff
@@ -64,14 +65,19 @@ func (a *Aster) PackageComment() string {
 	packageComments := []string{}
 
 	for pkgName, pkg := range a.packages {
+		var b strings.Builder
 		if len(a.packages) > 1 {
-			packageComments = append(packageComments, pkgName+":\n")
+			b.WriteString(pkgName + ":\n")
 		}
 		for _, f := range pkg.Files {
-			packageComments = append(packageComments, f.Doc.Text())
+			b.WriteString(f.Doc.Text())
 		}
+
+		packageComments = append(packageComments, b.String())
 	}
 
+	sort.Strings(packageComments)
```

Comment:
☝️ Previously, because Go does not guarantee the order of map keys when ranging over a map, this method could return package comments in an inconsistent order, which in turn caused flaky test failures given that the tests assume the package comments appear in a consistent order (In other words, `PackageComment()` would sometimes return comments in which `aster:*` appeared first, while other times `aster_test:*` comments would appear first). This seeks to fix all that by sorting the `packageComments` `[]string{}`.